Test/git-issues/git-issue-505.dfy

```diff
@@ -7,23 +7,23 @@ lemma a()
   var testbv: bv64 := a as bv64;
   var testval: int := testbv as int;
 
-  assert testval == a; // "Timed out on: assertion violation"
+  // assert testval == a; // "Timed out on: assertion violation"
```

Comment:
For the changes in this file, see also https://github.com/dafny-lang/dafny/pull/868/commits/e62b6e23d82d8eefe0799dfaba5b46e7fb8a88d1, which also had to adjust this test. Perhaps this test file should be removed altogether?

Reply:
Fixed -- just including the small BV tests, which do not timeout